stock_dynamic_routing/models/stock_move.py

```diff
@@ -242,7 +242,9 @@ def _routing_splits(self, moves_routing):
                     new_move_vals = move._split(qty)
                 if new_move_vals:
                     new_move = self.env["stock.move"].create(new_move_vals)
-                    new_move._action_confirm(merge=False)
+                    new_move.with_context(
```

Comment:
That one is maybe not required, I blocked the automatic reservation on all `_action_confirm` calls.